validation/src/test/scala/hmda/validation/engine/ts/syntactical/TsSyntacticalEngineSpec.scala

```diff
@@ -98,21 +82,15 @@ class TsSyntacticalEngineSpec extends PropSpec with PropertyChecks with MustMatc
   protected def failGenTs(badTs: Gen[TransmittalSheet]): Assertion = {
     badTs.sample match {
       case Some(x) =>
-        val fValidated = checkSyntactical(x, ValidationContext(None))
-        whenReady(fValidated) { validated =>
-          validated.isFailure mustBe true
-        }
+        checkSyntactical(x, ValidationContext(None)).isFailure mustBe true
```

Comment:
I believe you can use `mustBe a[Failure]` here instead, which would give more sensible error messages.